crates/fluvio-cli/src/connector/delete.rs

```diff
@@ -21,10 +21,25 @@ pub struct DeleteManagedConnectorOpt {
     name: String,
 }
 
+use fluvio::FluvioError;
+use fluvio_sc_schema::ApiError;
+use fluvio_sc_schema::errors::ErrorCode;
+
 impl DeleteManagedConnectorOpt {
     pub async fn process(self, fluvio: &Fluvio) -> Result<(), CliError> {
         let admin = fluvio.admin().await;
-        admin.delete::<ManagedConnectorSpec, _>(&self.name).await?;
+
+        admin
+            .delete::<ManagedConnectorSpec, _>(&self.name)
+            .await
+            .map_err(|e| match e {
+                FluvioError::AdminApi(ApiError::Code(ErrorCode::ManagedConnectorNotFound, _)) => {
+                    CliError::InvalidConnector(format!("{} not found", &self.name))
+                }
+                // Raise a bug for any Unhandled error.
```

Comment:
I agree that we need to properly handle all error messages correctly but this is probably not the preferred approach.
* First, BUG is a serious allegation that may or may not be correct for any unhandled. 
* We need to fix systematic not in a single isolated code.